content/docs/install/manual_demo.md

```diff
@@ -54,18 +54,16 @@ This command enables
 [Prometheus](https://github.com/prometheus/prometheus),
 [Grafana](https://github.com/grafana/grafana), and
 [Jaeger](https://github.com/jaegertracing/jaeger) integrations.
-The `OpenServiceMesh.enablePermissiveTrafficPolicy` chart value instructs OSM to ignore any policies and
```

Comment:
This is correct, but referring to the chart parameter in the `values.yaml` file. I think we should clarify this.